arcgis-ios-sdk-samples/Maps/Create and save a map/SaveAsViewController.swift

```diff
@@ -16,51 +16,37 @@ import UIKit
 
 protocol SaveAsVCDelegate: AnyObject {
     func saveAsViewController(_ saveAsViewController: SaveAsViewController, didInitiateSaveWithTitle title: String, tags: [String], itemDescription: String)
-    func saveAsViewControllerDidCancel(_ saveAsViewController: SaveAsViewController)
```

Comment:
My belief is that samples should be a simple as possible, especially when it comes to boilerplate features. The callback adds an unnecessary layer of complexity since the sample doesn't require cancel customization.

Reply:
> My belief is that samples should be a simple as possible

Being simple can be good, but being overly simplistic is not so good. A new implementation? Sure, it would be simpler to not create a cancel delegate callback. But in this case the delegate callback was already in place, so it didn't add any value to get rid of it. I would argue that it took away value by removing usage of a best practice.

Reply:
I disagree that this is a best practice in this scenario.

Reply:
Anyway, if you really want me to change it back I will. Marking this as resolved for now though.